frontend/model/contracts/shared/constants.js

```diff
@@ -118,7 +120,6 @@ export const POLL_TYPES = {
 }
 
 export const POLL_STATUS = {
-  ACTIVE: 'active', // the poll is open
-  CLOSED: 'closed', // everyone has voted on the poll
-  EXPIRED: 'expired' // not everyone has voted on the poll but it has expired
```

Comment:
While developing, I realised it doesn't make sense to close a poll as soon as everyone has voted because voters should be able to change their votes for some time until it expires. so the status values are either `ACTIVE` & `CLOSED`.